cscore/src/main/java/edu/wpi/first/cscore/OpenCvLoader.java

```diff
@@ -46,7 +46,13 @@ private Helper() {}
         RuntimeLoader.loadLibrary(Core.NATIVE_LIBRARY_NAME);
       } catch (IOException ex) {
         ex.printStackTrace();
-        System.exit(1);
+        try {
+          // Try adding a debug postfix: https://github.com/wpilibsuite/allwpilib/pull/6928
```

Comment:
No need to add a comment referring back to this PR.  That’s what git history is for.